src/app/components/DoctorCard/DoctorCard.stories.tsx

```diff
@@ -23,3 +25,18 @@ export const Card = (): JSX.Element => (
     city="Hamburg"
   />
 );
+
+export const CardWithAction = (): JSX.Element => (
+  <DoctorCard
+    imgPath={imageFile}
+    heading="Dipl.-Psych. Sabine Wery von Limont"
+    adress="Max-Brauer-Alee 52,"
+    plz="22765"
+    city="Hamburg"
+    children={
+      <>
+        <ActionButton className={styles.actionButton} children="zum Profile" />
+      </>
+    }
+  />
```

Comment:
Instead of passing the button via the `children` prop like you did, I'd suggest doing it like this.
Or even better, why not make the Button part of the card and optionally displayed with `onClick` and all that
```suggestion
  <DoctorCard
    imgPath={imageFile}
    heading="Dipl.-Psych. Sabine Wery von Limont"
    adress="Max-Brauer-Alee 52,"
    plz="22765"
    city="Hamburg" 
  />
     <ActionButton className={styles.actionButton} children="zum Profile" />
  </DoctorCard>
```